audio/param.rs

```diff
@@ -327,9 +327,10 @@ impl AutomationEvent {
             AutomationEvent::SetValueCurveAtTime(_, start, _) => start,
             AutomationEvent::RampToValueAtTime(_, _, tick) => tick,
             AutomationEvent::SetTargetAtTime(_, start, _) => start,
-            AutomationEvent::CancelAndHoldAtTime(t) => t,
-            AutomationEvent::CancelScheduledValues(..) | AutomationEvent::SetValue(..) => {
-                unreachable!("CancelScheduledValues/SetValue should never appear in the timeline")
+            AutomationEvent::CancelAndHoldAtTime(tick) => tick,
+            AutomationEvent::CancelScheduledValues(tick) => tick,
+            AutomationEvent::SetValue(..) => {
+                unimplemented!("SetValue should never appear in the timeline")
```

Comment:
One thing I was not sure of is whether ```SetValue``` is something that is *actually* supposed to be unreachable or if, like ```CancelScheduledValues```, it was just not implemented yet.